jobs/build/build-sync/build.groovy

```diff
@@ -97,61 +92,32 @@ release:gen-multiarch-payload
 	sh("mv src_dest* ${mirrorWorking}/")
 	sh("mv image_stream*.yaml ${mirrorWorking}/")
 	artifacts.addAll(["MIRROR_working/src_dest*", "MIRROR_working/image_stream*"])
-    } else {
-	echo("Generating SRC=DEST for one arch")
-	buildlib.doozer """
-${images}
---working-dir "${mirrorWorking}" --group 'openshift-${params.BUILD_VERSION}'
-release:gen-payload
---src-dest ${ocMirrorInput}
---image-stream ${ocIsObject}
---is-base ${baseImageStream}
-'${ocFmtStr}'
-"""
-	echo("Generated file:")
-	echo("######################################################################")
-	echo("${ocMirrorInput}")
-	sh("cat ${ocMirrorInput}")
-    }
 }
 
 // ######################################################################
 // Now run the actual mirroring commands. We wrap this in a retry()
 // loop because it is known to fail occassionally depending on the
 // health of the source/destination endpoints.
 def buildSyncMirrorImages() {
-    if ( multiarch ) {
-        for ( String arch: arches ) {
-            retry ( 3 ) {
-                echo("Attempting to mirror arch: ${arch}")
-                // Always login again. It may expire between loops
-                // depending on amount of time elapsed
-                buildlib.registry_quay_dev_login()
-                buildlib.oc "${logLevel} image mirror ${dryRun} --filename=${mirrorWorking}/src_dest.${arch} --filter-by-os=${arch}"
-            }
-        }
-    } else {
+    for ( String arch: arches ) {
         retry ( 3 ) {
+            echo("Attempting to mirror arch: ${arch}")
+            // Always login again. It may expire between loops
+            // depending on amount of time elapsed
             buildlib.registry_quay_dev_login()
-            buildlib.oc "${logLevel} image mirror ${dryRun} --filename=${ocMirrorInput}"
-            currentBuild.description += "\nMirrored: default arch (amd64)"
+            buildlib.oc "${logLevel} image mirror ${dryRun} --filename=${mirrorWorking}/src_dest.${arch}"
```

Comment:
Is `--filter-by-os` missing or something related to the command changed?

Reply:
The entries in the file are already arch specific at this point. There should be nothing to filter.